src/send_future.rs

```diff
@@ -70,8 +70,11 @@ where
     }
 }
 
-/// "Sending" state of [`SendFuture`] for cases where the actor type is named.
-pub struct ActorNamedSending<A, Rc: RefCounter>(Sending<A, Rc>);
+/// "Sending" state of [`SendFuture`] for cases where the actor type is named and we sent a single message.
+pub struct ActorNamedSending<A, Rc: RefCounter>(Sending<A, MessageToOne<A>, Rc>);
+
+/// "Sending" state of [`SendFuture`] for cases where the actor type is named and we broadcast a message.
+pub struct ActorNamedBroadcasting<A, Rc: RefCounter>(Sending<A, MessageToAll<A>, Rc>);
```

Comment:
I am not too happy with the terminology that we use to differentiate between regular messages and broadcasts. We a bit of a mix here throughout the codebase.

That said, I don't think this PR makes it any worse other than maybe surfacing this problem even more.